### ISSUE TITLE
Better Button Locations in OpenID Connect Client Configuration

### ISSUE THREAD

**Issue by afroDC** [GluuFederation/oxTrust#1011]
Currently the buttons for configuration settings like `Add Scope` are all mashed together at the bottom of the page. This doesn't work well intuitively and is a hindrance when configuring clients. See below:

![gluu_openidc_client_buttons](https://user-images.githubusercontent.com/30447907/41371492-1f5b3ce4-6f10-11e8-86df-6a877b2eb8ef.png)

The buttons should be under the corresponding field they're populating:

![gluu_openidc_client_page](https://user-images.githubusercontent.com/30447907/41372125-f9e4f764-6f11-11e8-8318-1dc30ee95b5f.png)
Also the `Default requested Authentication Context Class Reference values:` field description should read `Default requested Authentication Context Class Refernce (ACR) values:` for clarity.

**Comment by willow9886** [comment]
@shekhar16 please use your best judgement for implementing this in a user friendly way... that means spacing should be visually pleasing, design should be responsive, popups should function well, etc.

**Comment by shekhar16** [comment]
![image](https://user-images.githubusercontent.com/15937500/41496191-40b017ea-7157-11e8-908a-a0b26bd25434.png)

**Comment by natt-tester** [comment]
@shekhar16 Please check if the commit has been merged to 3.1.4. I have just checked dev6 with a new war and the buttons are still mashed together:

![2018-08-10_19h38_25](https://user-images.githubusercontent.com/38302725/43951023-56741f46-9cd5-11e8-87f4-4690119a126a.png)

**Comment by afroDC** [comment]
They definitely didn't make it into the 3.1.4 repo. Please take care of this @shekhar16

**Comment by shekhar16** [comment]
@natt-tester  @afroDC  yes it is not merged in any of our main branch in repo. Let me check with Yuriy
